components/rating/rating.html

```diff
@@ -0,0 +1,159 @@
+            <li class="components-list-item">
+              <a href="../avatar/avatar.html">Avatar</a>
+            </li>
+            <li class="components-list-item">
+              <a href="../alert/alert.html">Alert</a>
+            </li>
+            <li class="components-list-item">
+              <a href="../badge/badge.html">Badge</a>
+            </li>
+            <li class="components-list-item">
+              <a href="../button/button.html">Button</a>
+            </li>
+            <li class="components-list-item">
+              <a href="../card/card.html">Cards</a>
+            </li>
+            <li class="components-list-item">
+              <a href="../image/image.html">Image</a>
+            </li>
+            <li class="components-list-item">
+              <a href="../input/input.html">Input</a>
+            </li>
+            <li class="components-list-item">
+              <a href="../typography/typography.html">Typography</a>
+            </li>
+            <li class="components-list-item">
+              <a href="../lists/lists.html">Lists</a>
+            </li>
+            <li class="components-list-item">
+              <a href="../typography/typography.html">Navigation</a>
+            </li>
+
+            <li class="components-list-item">
+              <a href="../typography/typography.html">Modal</a>
+            </li>
+            <li class="components-list-item">
+              <a href="../toast/toast.html">Toast</a>
+            </li>
+            <li class="components-list-item">
+              <a href="../rating/rating.html" id="active-page">Rating</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+
+      <!-- MAIN SECTION -->
+
+      <section class="main" id="main-section">
+        <h1 class="main-heading">RATING</h1>
+        <p>
+          Ratings help users decide whether the product or the service is good
+          enough for them or not.
+        </p>
+
+        <!-- Read Only Ratings -->
+        <h2>Read only Ratings</h2>
+        <p>
+          Read only ratings displays ratings to the users regarding specific
+          product or service.
+        </p>
+        <!-- Code Output -->
+        <div class="code-output-container">
+          <div class="rating-read-container">
+            <i class="fas fa-star fa-lg rating-checked"></i>
+            <i class="fas fa-star fa-lg rating-checked"></i>
+            <i class="fas fa-star fa-lg rating-checked"></i>
+            <i class="fas fa-star fa-lg rating-checked"></i>
+            <i class="fas fa-star fa-lg"></i>
+          </div>
+        </div>
+
+        <!-- CODE -->
+        <div class="code-example-container">
+          <iframe
+            src="https://www.thiscodeworks.com/embed/6200ea264dd44f0015f3cd94"
+            style="width: 100%; height: 177px"
+            frameborder="0"
+          ></iframe>
+        </div>
+      </section>
+    </div>
+
+    <script src="./rating.js"></script>
```

Comment:
Couldn't see its implementation on the main site. So is it required?You can remove it if not.

Reply:
I will remove it